_posts/0000-01-05-russa011.md

```diff
@@ -6,4 +6,4 @@ title: "Welcome to our slide deck!"
 ![fry](https://cloud.githubusercontent.com/assets/16547949/25400891/fb18941a-29c1-11e7-82b9-35e589443a60.jpg)
```

Comment:
Change the title to something different on line 3

Reply:
Done - pls approve. :-)